src/assets/sass/v2/blocks/_space.scss

```diff
@@ -0,0 +1,6 @@
+.space--none {
```

Comment:
Would govuk-!-margin-bottom-0 from the kit work instead of adding our own space classes?